packages/react-core/src/components/Tabs/examples/TabsHelp.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { Tabs, Tab, TabTitleText, TabAction, Popover } from '@patternfly/react-core';
+import HelpIcon from '@patternfly/react-icons/dist/esm/icons/help-icon';
+
+export const TabsHelp: React.FunctionComponent = () => {
+  const [activeTabKey, setActiveTabKey] = React.useState<number>(0);
+
+  const tabs = ['Users', 'Containers', 'Database', 'Disabled', 'ARIA disabled', 'Help disabled'];
+
+  const helpPopover = (header: string, popoverRef: React.RefObject<any>) => (
+    <Popover
+      headerContent={<div>{header}</div>}
+      bodyContent={
+        <div>
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam id feugiat augue, nec fringilla turpis.
+        </div>
+      }
+      footerContent="Popover footer"
+      reference={popoverRef}
+    />
+  );
+
+  return (
+    <Tabs
+      activeKey={activeTabKey}
+      onSelect={(event: any, tabIndex: string | number) => setActiveTabKey(tabIndex as number)}
+      aria-label="Tabs in the help action example"
+      role="region"
+    >
+      {tabs.map((tab, index) => {
+        const ref = React.createRef<HTMLElement>();
+
+        return (
+          <Tab
+            key={index}
+            eventKey={index}
+            aria-label={`Help action content - ${tab}`}
+            title={<TabTitleText>{tab}</TabTitleText>}
+            {...(tab === 'Disabled' && { isDisabled: true })}
+            {...(tab === 'ARIA disabled' && { isAriaDisabled: true })}
+            actions={
+              <>
+                <TabAction
+                  aria-label={`Help action for ${tab}`}
+                  ref={ref}
```

Comment:
These don't have `isHelpAction` - is that intentional? I'm curious if we should render this for the user so they don't have to construct it themselves. WDYT?

Reply:
I'll add `isHelpAction`, must have forgotten it

Reply:
Or should `isHelpAction` be the default? LMK and I can re-update, but adding the flag to each for now.

Reply:
I see `<TabAction>` being something a user could define as they see fit, so I don't think `isHelpAction` should be the default, but I could be missing some other kind of advantage that would provide.

Reply:
You can still define `help - foo - bar - close` all in the `actions` currently.